VAMobile/src/store/slices/secureMessagingSlice.ts

```diff
@@ -519,7 +519,7 @@ export const sendMessage =
       if (isErrorObject(error)) {
         logNonFatalErrorToFirebase(error, `sendMessage: ${secureMessagingNonFatalErrorString}`)
         dispatch(dispatchFinishSendMessage(error))
-        showSnackBar(messages.errorMsg, dispatch, retryFunction, false, true)
+        replyToID && hasErrorCode(SecureMessagingErrorCodesConstants.TRIAGE_ERROR, error) ? '' : showSnackBar(messages.errorMsg, dispatch, retryFunction, false, true)
```

Comment:
Since the empty string (`''`) isn't doing anything, the triage error logic could be stored in a variable, and you could do something like:

```ts
const isReplyWithTriageError = replyToID && hasErrorCode(SecureMessagingErrorCodesConstants.TRIAGE_ERROR, error)
!isReplyWithTriageError && showSnackBar(messages.errorMsg, dispatch, retryFunction, false, true)
```

Reply:
Fixed as suggested.